tt_metal/impl/kernels/kernel.cpp

```diff
@@ -173,8 +174,13 @@ std::pair<uint64_t, uint64_t> DataMovementKernel::get_runtime_args_range() const
 }
 
 std::pair<uint64_t, uint64_t> EthernetKernel::get_runtime_args_range() const {
+    std::pair<uint64_t, uint64_t> arg_base_to_result_base;
     // TODO: get this from eth l1 map
-    std::pair<uint64_t, uint64_t> arg_base_to_result_base = {eth_l1_mem::address_map::ERISC_L1_ARG_BASE, eth_l1_mem::address_map::ERISC_APP_RESERVED_BASE};
+    if (this->config_.eth_mode == Eth::IDLE) {
+        arg_base_to_result_base = {IDLE_ERISC_L1_ARG_BASE, IDLE_ERISC_L1_RESULT_BASE};
```

Comment:
ah I see why you added `_RESULT_BASE` but I think we should just define `IDLE_ERISC_L1_ARG_BASE` and `IDLE_ERISC_RT_ARG_SIZE` or something like that